src/main/java/com/example/sccshop/payment/PaymentResult.java

```diff
@@ -0,0 +1,30 @@
+package com.example.sccshop.payment;
+
+/**
+ * @author Olga Maciaszek-Sharma
+ */
+public class PaymentResult {
+
+	private String uuid;
+	private PaymentStatus status;
+
+	public void setUuid(String uuid) {
+		this.uuid = uuid;
+	}
+
+	public void setStatus(PaymentStatus status) {
+		this.status = status;
+	}
+
+	String getUuid() {
+		return uuid;
+	}
+
+	boolean successful() {
+		return PaymentStatus.SUCCESS.equals(status);
+	}
+
+	enum PaymentStatus {
+		SUCCESS, FAILED
```

Comment:
it should be `FAILURE` due to the changes in https://github.com/eddumelendez/scc-payment/pull/6

Reply:
yes, I will merge it to the most recent master.